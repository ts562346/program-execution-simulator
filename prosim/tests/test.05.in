test 3
BLOCK 3
DOOP 5
HALT

