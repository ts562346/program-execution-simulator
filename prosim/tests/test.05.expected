00000: BLOCK
00003: DOOP
00008: HALT

DOOP count : 1
DOOP time  : 5
BLOCK count: 1
BLOCK time : 3
